Src/Sentry.Xamarin/Internals/Device/Screenshot/ScreenshotAttachmentContent.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.IO;
+using Sentry.Extensibility;
+using Xamarin.Essentials;
+using static Xamarin.Essentials.Screenshot;
+
+namespace Sentry.Internals.Device.Screenshot
+{
+    internal class ScreenshotAttachmentContent : IAttachmentContent
+    {
+        private SentryXamarinOptions _options { get; }
+        public ScreenshotAttachmentContent(SentryXamarinOptions options) => _options = options;
+
+        public Stream GetStream()
+        {
+            try
+            {
+                var screenStream = CaptureAsync().ConfigureAwait(false).GetAwaiter().GetResult();
+                return screenStream.OpenReadAsync(ScreenshotFormat.Jpeg).ConfigureAwait(false).GetAwaiter().GetResult();
```

Comment:
Since you're not actually switching contexts, do you still need `ConfigureAwait(false)` here?

Reply:
We have some roslyn analyzer that warns if we don't use it on any await. Since it's library dev we just error on the side of adding the boiler plate and avoiding the overhead of thinking through if it applies or not (is there a sync context) and risk of causing deadlocks